conf/apt-packages.txt

```diff
@@ -15,7 +15,6 @@ libxslt1-dev
 libncurses5-dev
 libjpeg-dev
 gdal-bin
-uwsgi
```

Comment:
It makes more sense to install uWSGI with pip as it comes packaged with the python plugin, plus we get a newer version.